packages/material-ui/src/Popper/Popper.js

```diff
@@ -293,6 +293,7 @@ Popper.propTypes = {
 
 Popper.defaultProps = {
   disablePortal: false,
+  keepMounted: false,
```

Comment:
This would conflict with https://next.material-ui.com/guides/api/#property-naming and be a breaking change. 

This change should be discussed separately.

Reply:
@eps1lon I don't follow. It would be a breaking change and goes against the convention if we were doing `keepMounted: true,`. I'm doing the opposite. It matches the Modal.

Reply:
It fixes a minor documentation neglect.

Reply:
https://github.com/mui-org/material-ui/blob/a243f67a4406d77b038f429fcd85c2d1a49d4160/packages/material-ui/src/Modal/Modal.js#L413

Reply:
Yeah the first part was nonsense. 

Better docs make sense. We're not documenting it for the Button either though. Might have some other offenders. That use `undefined` as `false` inside the component. Not a real issue IMO.

Reply:
> We're not documenting it for the Button either though. Might have some other offenders.

Which props?